CITATIONS.md

```diff
@@ -0,0 +1,55 @@
+
+## [Nextflow](https://www.ncbi.nlm.nih.gov/pubmed/28398311/)
+
+> Di Tommaso P, Chatzou M, Floden EW, Barja PP, Palumbo E, Notredame C. Nextflow enables reproducible computational workflows. Nat Biotechnol. 2017 Apr 11;35(4):316-319. doi: 10.1038/nbt.3820. PubMed PMID: 28398311.
+
+## Pipeline tools
+
+* [BEDTools](https://www.ncbi.nlm.nih.gov/pubmed/20110278/)
+  > Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010 Mar 15;26(6):841-2. doi: 10.1093/bioinformatics/btq033. Epub 2010 Jan 28. PubMed PMID: 20110278; PubMed Central PMCID: PMC2832824.
+
+* [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
+
+* [featureCounts](https://www.ncbi.nlm.nih.gov/pubmed/24227677/)
+  > Liao Y, Smyth GK, Shi W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014 Apr 1;30(7):923-30. doi: 10.1093/bioinformatics/btt656. Epub 2013 Nov 13. PubMed PMID: 24227677.
+
+* [MultiQC](https://www.ncbi.nlm.nih.gov/pubmed/27312411/)
+  > Ewels P, Magnusson M, Lundin S, Käller M. MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics. 2016 Oct 1;32(19):3047-8. doi: 10.1093/bioinformatics/btw354. Epub 2016 Jun 16. PubMed PMID: 27312411; PubMed Central PMCID: PMC5039924.
+
+* [SAMtools](https://www.ncbi.nlm.nih.gov/pubmed/19505943/)
+  > Li H, Handsaker B, Wysoker A, Fennell T, Ruan J, Homer N, Marth G, Abecasis G, Durbin R; 1000 Genome Project Data Processing Subgroup. The Sequence Alignment/Map format and SAMtools. Bioinformatics. 2009 Aug 15;25(16):2078-9. doi: 10.1093/bioinformatics/btp352. Epub 2009 Jun 8. PubMed PMID: 19505943; PubMed Central PMCID: PMC2723002.
+
+* [StringTie2](https://www.ncbi.nlm.nih.gov/pubmed/31842956/)
+  > Kovaka S, Zimin AV, Pertea GM, Razaghi R, Salzberg SL, Pertea M. Transcriptome assembly from long-read RNA-seq alignments with StringTie2 Genome Biol. 2019 Dec 16;20(1):278. doi: 10.1186/s13059-019-1910-1. PubMed PMID: 31842956; PubMed Central PMCID: PMC6912988.
+
+* [UCSC tools](https://www.ncbi.nlm.nih.gov/pubmed/20639541/)
+  > Kent WJ, Zweig AS, Barber G, Hinrichs AS, Karolchik D. BigWig and BigBed: enabling browsing of large distributed datasets. Bioinformatics. 2010 Sep 1;26(17):2204-7. doi: 10.1093/bioinformatics/btq351. Epub 2010 Jul 17. PubMed PMID: 20639541; PubMed Central PMCID: PMC2922891.
+
+## R packages
+
+* [R](https://www.R-project.org/)
+  > R Core Team (2017). R: A language and environment for statistical computing. R Foundation for Statistical Computing, Vienna, Austria.
+
+* [DESeq2](https://www.ncbi.nlm.nih.gov/pubmed/25516281/)

```

Comment:
would you want to add dexseq as well? 
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3460195/
If needed, you can also cite bambu using zenodo doi:
Chen Ying, jonathangoeke, & Yuk Kei Wan. (2020, June 18). GoekeLab/bambu: bambu release version 0.2.0 (Version v0.2.0). Zenodo. http://doi.org/10.5281/zenodo.3900025

Reply:
Yep! Definitely! Was going to ask if you could add those in please after this PR is merged 😁